salsah/src/public/vendor/ckeditor-4.7.2-Editor/adapters/jquery.js

```diff
@@ -0,0 +1,10 @@
+﻿/*
+ Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
```

Comment:
could this collide with the jquery version we have already in SALSAH?

Reply:
we don't use that feature (integration of CKEditor on any textarea):  
https://docs.ckeditor.com/#!/guide/dev_jquery

it could collide in two ways:
- the version of jQuery that we use is not compatible and this feature would not work... we will find out if we use it
- there are two concurrent version of jQuery, it is not the case as CKEditor doesn't reload the jquery, it uses the one that is already loaded